examples/FaxSend.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Dropbox.Sign.Api;
+using Dropbox.Sign.Client;
+using Dropbox.Sign.Model;
+
+public class Example
+{
+    public static void Main()
+    {
+        var config = new Configuration();
+        config.Username = "YOUR_API_KEY";
+
+        var faxApi = new FaxApi(config);
+
+        var data = new FaxCreateRequest(
+            file: "<form-data file upload>",
```

Comment:
`file` to be renamed to `files`

```suggestion
            file: files,
```